### PR TITLE
feat(tooltip): support error tooltip position

### PR DESCRIPTION
**Pull request by megansmith-box**

### CODE REVIEW

File: src/components/select-button/SelectButton.js [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
     error?: React.Node,
+    errorTooltipPosition?: Position,
     isDisabled: boolean,
 };
 
 const SelectButton = React.forwardRef<Props, HTMLButtonElement>(
-    ({ children, className = '', error, isDisabled = false, ...rest }: Props, ref) => (
-        <Tooltip isShown={!!error} position="middle-right" text={error} theme="error">
+    ({ children, className = '', error, errorTooltipPosition, isDisabled = false, ...rest }: Props, ref) => (
+        <Tooltip isShown={!!error} position={errorTooltipPosition || 'middle-right'} text={error} theme="error">
```
Comment (tjuanitas) [inline src/components/select-button/SelectButton.js:19]: I see we do `position={errorTooltipPosition || 'middle-right'}` in other places as well but i'm curious why not we don't default `errorTooltipPosition` in the function parameters

File: src/components/select-button/SelectButton.js [hunk 2]
```diff
@@ -2,19 +2,21 @@
 import * as React from 'react';
 import classNames from 'classnames';
 
+import type { Position } from '../tooltip';
```
Comment (tjuanitas) [inline src/components/select-button/SelectButton.js:5]: minor but could move this after the Tooltip component

**Review by tjuanitas — APPROVED** [review]

File: src/components/select-button/SelectButton.js (hunk 1 above)
Comment (megansmith-box) [reply src/components/select-button/SelectButton.js:19]: No reason in particular, I see it both ways in the code. I like the default function parameter suggestion!

**Review by tjuanitas — APPROVED** [review]

**Review by QingyuChai — DISMISSED** [review]
lgtm! I think one of Trevor's comments is still unaddressed though

this one: https://github.com/box/box-ui-elements/pull/3187#discussion_r1052737897

**Review by QingyuChai — APPROVED** [review]
thx for the fix!

**Review by tjuanitas — APPROVED** [review]

**Pull request merged**